packages/minter-contracts/ligo/src/fixed_price_sale/fixed_price_sale_market.mligo

```diff
@@ -66,6 +78,15 @@ let get_sale(sale_id, storage: sale_id * storage) : sale =
     | None -> (failwith "NO_SALE": sale)
     | Some s -> s)
 
+let tx_price_and_fee (money_token_id, money_token_address, fee, fee_address, sale_price, seller, oplist :
```

Comment:
Consider giving more descriptive name to `fee_address` parameter.
Something like `fee_recepient_address`